spanishdict/equivalency
# Feature 165723543 edit distance

@sandinmyjoints 

TODO:
- [ ] Follow up PR updating Equivalency usage docs

### What

https://www.pivotaltracker.com/story/show/165723543

Add damerau-levenshtein edit distance algorithm to equivalency. 

https://www.npmjs.com/package/damerau-levenshtein

### How to test

- Take a look at the logical structure of the test suite and see if it makes sense
     - `jest --watch`
- Take a look at the test cases and make sure you don't see any discrepancies
     - The tests that I actually made are within the 2 `editDistance` describe blocks

Test cases come from the Epic Doc (https://www.notion.so/curiositymedia/Vocab-Quiz-Accuracy-Leniency-9b722f8e7edf473798ddb2b921e39e68#3edd5ca42765473489a7a389da4f5902), but I added some of my own as well. 

cc @spanishdict/engineering-sd-engage
